sommelier/chain.json

```diff
@@ -29,9 +29,9 @@
   },
   "codebase": {
     "git_repo": "https://github.com/PeggyJV/sommelier",
-    "recommended_version": "v5.0.0",
```

Comment:
Can we please also create a new object at the bottom of the versions array for this v6?

Reply:
Apologies for overlooking that. Resubmitting with proper changes now.

Reply:
looks like it's still removing v5 even though we want to keep that

Reply:
Apologies if I'm not following- if the current chain version is v6, why do we keep v5 on here?

Reply:
to keep a record of versions. Ideally we have all versions in here, with their effective block height.